airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/load/message/DestinationMessage.kt

```diff
@@ -79,15 +82,33 @@ data class Meta(
     }
 }
 
-sealed interface DestinationRecord : DestinationRecordDomainMessage
+sealed interface DestinationRecord : DestinationRecordDomainMessage {
+    val serialized: String
+}
+
+/**
+ * Represents a record already in its serialized state. The intended use is for conveying records
+ * from stdin to the spill file, where reserialization is not necessary.
+ */
+data class DestinationRecordSerialized(
+    override val stream: DestinationStream.Descriptor,
+    override val serialized: String
+) : DestinationRecord {
+    override fun asProtocolMessage(): AirbyteMessage {
```

Comment:
just curious: is this actually needed anywhere? or is it only to fulfill the interface?

Reply:
It's just to fulfill the interface. I want to rip all of this out that isn't production-necessary and move it to testing